model/_init.cf

```diff
@@ -52,7 +52,7 @@ entity Host extends std::Host:
         :attr remote_user: The remote user for the remote agent to login with
         :attr remote_port: The remote port for this remote agent to use.
     """
-    ip::ip ip
+    std::ipv_any_address ip
```

Comment:
Should this be `ipv4_address`? The ticket says so but that seems restrictive to me. Keep in mind that I will use the same type in the std module.

Reply:
I think supporting ipv6 is fine